experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts

```diff
@@ -542,6 +557,7 @@ export class SharedPropertyTree extends SharedObject {
 				if (
 					snapshotSummary.remoteChanges === undefined ||
 					snapshotSummary.remoteTipView === undefined ||
+                    snapshotSummary.remoteHeadGuid === undefined ||
```

Comment:
We should remove this exception. This will cause old summaries to become invalid and thus will prevent clients with different versions to interoperate.